cg/store/api/status.py

```diff
@@ -545,17 +551,27 @@ def _all_samples_have_sequence_data(links: List[models.FamilySample]) -> bool:
         )
 
     def analyses_to_upload(self, pipeline: Pipeline = None) -> List[models.Analysis]:
```

Comment:
A lot better but I think it could be even more atomic. You should split the query and filtering into separate functions with unit tests so that each query/filter is testable and reusable, see https://github.com/Clinical-Genomics/cg/pull/1545/files for an example on how did it for cases_to_analyze.